com.unity.multiplayer.mlapi/Runtime/Core/SnapshotSystem.cs

```diff
@@ -20,16 +20,16 @@ internal struct VariableKey
         public ulong NetworkObjectId; // the NetworkObjectId of the owning GameObject
         public ushort BehaviourIndex; // the index of the behaviour in this GameObject
         public ushort VariableIndex; // the index of the variable in this NetworkBehaviour
+        public ushort TickWritten; // the network tick at which this variable was set
     }
 
     // Index for a NetworkVariable in our table of variables
     // Store when a variable was written and where the variable is serialized
     internal struct Entry
     {
         public VariableKey Key;
-        public ushort TickWritten; // the network tick at which this variable was set
         public ushort Position; // the offset in our Buffer
-        public ushort Length; // the length of the data in Buffer
+        public ushort Length; // the Length of the data in Buffer
         public bool Fresh; // indicates entries that were just received
 
         public const int NotFound = -1;
```

Comment:
Should this live inside the actual struct?

Reply:
I don't get your question. Maybe the comment can be improved. This is the length of a specific entry. So, yes, each Entry requires its own length.

Reply:
Ups, I meant the NotFound constant variable.

Reply:
It could be in Snapshot too. But since it is a constant and doesn't contribute to object size, what difference does it make? I kind-of like`Entry.NotFound` as it expresses clearly what it means. How do others feel ?